app/routes/cdo/manage/tasks/generic-task-helper.js

```diff
@@ -0,0 +1,76 @@
+const { tasks } = require('../../../../constants/cdo/index')
+const ViewModel1 = require('../../../../models/cdo/manage/tasks/send-application-pack')
+const ViewModel2 = require('../../../../models/cdo/manage/tasks/record-insurance-details')
+const ViewModel3 = require('../../../../models/cdo/manage/tasks/record-microchip-number')
+const ViewModel4 = require('../../../../models/cdo/manage/tasks/record-application-fee-payment')
+const ViewModel5 = require('../../../../models/cdo/manage/tasks/send-form2')
+const ViewModel6 = require('../../../../models/cdo/manage/tasks/record-verification-dates')
+const { validateSendApplicationPack } = require('../../../../schema/portal/cdo/tasks/send-application-pack')
```

Comment:
wondering if it will clearer going forward for naming of ViewModel to be more in line with the type of view mode...e.g. ``ViewModelSendApplicationPack`` or ``ViewModel1SendApplicationPack``